hera_cal/abscal.py

```diff
@@ -3239,7 +3245,11 @@ def post_redcal_abscal_run(data_file, redcal_file, model_files, raw_auto_file=No
                         if not np.all(list(flags.values())):
                             # load model and rephase
                             model_times_to_load = [d2m_time_map[time] for time in hd.times[tinds]]
-                            model, model_flags, _ = io.partial_time_io(hdm, model_times_to_load, bls=model_bl_to_load)
+                            model, model_flags, _ = io.partial_time_io(hdm, np.unique(model_times_to_load), bls=model_bl_to_load)
```

Comment:
It may be worth the effort to not use `np.unique` here, since `np.unique` sorts the array it acts on. This has certainly caused problems with the validation work I'm doing, where the sim runs from 0-24 hrs LST linearly and the data has a phase wrap--the sorting done by `np.unique` means that the times are loaded in the wrong order, at least in my case (using the `select` method of the `HERAData` object).

Reply:
I suspect the easiest way to avoid this problem is to just do `model.select_or_expand_times(model_times_to_load)` regardless of whether times are repeated in `model_times_to_load`.

Reply:
The way this works under the hood, the order of the times that we're doing a select on doesn't matter, nor does the existence of duplicate times. I only put this here to be explicit that there might be duplicate times. Besides, this is acting on JDs not LSTs, so sorting would be OK anyway.